recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
+   md5: 951beb9a826c0c02fd6c08571b4365d2
+
+test:
+ command:
+  - run.sh test.vcf out logs k_out k_log 2 5 ab
```

Comment:
This is just something I experiment. I having troubles with the relative path inside the script since everything should be in one directory but I suggest me to put the snmf.sh in the bin so the relative path to call other files won't work right.  Do you have any idea to fix that